qdax/core/mels.py

```diff
@@ -0,0 +1,129 @@
+"""Core components of the MAP-Elites Low-Spread algorithm."""
+from __future__ import annotations
+
+from functools import partial
+from typing import Callable, Optional, Tuple
+
+import jax
+
+from qdax.core.containers.mels_repertoire import MELSRepertoire
+from qdax.core.emitters.emitter import Emitter, EmitterState
+from qdax.core.map_elites import MAPElites
+from qdax.types import (
+    Centroid,
+    Descriptor,
+    ExtraScores,
+    Fitness,
+    Genotype,
+    Metrics,
+    RNGKey,
+)
+
+
+def multi_eval_scoring_function(
```

Comment:
There is a file containing a similar function in qdax/utils/sampling.py, that is uses by ME-ES, maybe we want to uniform this? 
multi_eval_scoring_function() seems to be a subcase of sampling() that returns all the evaluations, while sampling returns the final average, so sampling() could just wrap multi_eval_scoring_function().

Reply:
Hi @manon-but-yes, thank you for the feedback! I have created a `multi_sample_scoring_function` in `sampling.py` and removed `multi_eval_scoring_function` from `mels.py`. I also changed `num_evals` to `num_samples` everywhere I could find it. Let me know what you think!

Reply:
It's perfect!